src/components/VideoEditor.tsx

```diff
@@ -0,0 +1,229 @@
+        <TextField
+          sx={{ '& .dsoQSg': { paddingTop: '10px' } }}
+          defaultValue={vidData?.channel?.link}
+          onChange={(e) => handleOnChangeChannel(e, "link")}
+          helperText="Optional: YouTube channel Link"
+          variant="filled"
+          size='small'
+          value={vidData?.channel?.link || ""}
+        />
+        <TextField
+          sx={{ '& .dsoQSg': { paddingTop: '10px' } }}
+          defaultValue={vidData?.startTimeS}
+          onChange={(e) => handleOnChange(e, "startTimeS")}
+          helperText="Optional: Video start time (seconds)"
+          variant="filled"
+          size='small'
+          value={vidData?.startTimeS || ""}
+        />
+        <TextField
+          sx={{ '& .dsoQSg': { paddingTop: '10px' } }}
+          defaultValue={vidData?.livestream ? "Yes" : "No"}
+          onChange={(e) => handleOnChange(e, "livestream")}
+          helperText="Optional: Yes / No livestream"
+          variant="filled"
+          size='small'
+          select
+          value={vidData?.livestream || ""}
+        >
+          <MenuItem key="true" value="true">Yes</MenuItem>
+          <MenuItem key="false" value="false">No</MenuItem>
+        </TextField>
+      </Stack>
+
+
+      {showDeleteAlert &&
+        <Stack>
+          <Alert
+            severity="error"
+            action={
+              <Stack direction="row">
+                <Button color="inherit" size="small" onClick={deleteAction}>YES</Button>
+                <Button color="inherit" size="small" onClick={() => setShowDeleteAlert(false)}>NO</Button>
+              </Stack>
+            }
+          >
+            <AlertTitle>Delete Video?</AlertTitle>
+            This cannot be undone
+          </Alert>
+        </Stack>
+      }
+
+      {showEditAlert &&
+        <Stack>
+          <Alert
+            severity="error"
+            action={
+              <Stack direction="row">
+                <Button color="inherit" size="small" onClick={editVideoAction}>YES</Button>
+                <Button color="inherit" size="small" onClick={() => setShowEditAlert(false)}>NO</Button>
+              </Stack>
+            }
+          >
+            <AlertTitle>Save Changes?</AlertTitle>
+            This action cannot be undone
+          </Alert>
+        </Stack>
+      }
+
+      {!addVideo &&
+        <Stack className="button-stack" direction="row" spacing={10}>
+          <Button onClick={verifyEdit} disabled={!!showEditAlert || !!showDeleteAlert} size='small' variant="contained" color='secondary'>Save Changes</Button>
+          <Button onClick={verifyDelete} disabled={!!showEditAlert || !!showDeleteAlert} variant="outlined" color="error">Delete Video</Button>
+        </Stack>
+      }
+
+      {!editVideo &&
+        <Stack className="button-stack" direction="row" spacing={10}>
+          <Button onClick={addVideoAction} size='small' variant="contained" color='secondary'>Add Video</Button>
+        </Stack>
+      }
```

Comment:
I think its more clear to do the opposite check here:
```
{addVideo &&
        <Stack className="button-stack" direction="row" spacing={10}>
          <Button onClick={addVideoAction} size='small' variant="contained" color='secondary'>Add Video</Button>
        </Stack>
      }

...
```
Basically show it if the prop is defined. Unless I am missing a reason why it must be done this way?